### ISSUE TITLE
CoffeeLint settings not being saved.

### ISSUE THREAD

**Issue by dlevs** [bdkjones/CodeKit2#442]
Hey Bryan, thanks for the great app!

My projects' CoffeeLint settings aren't being saved properly. If I change a setting for a project, such as changing indent from 2 to 4 spaces, this change will work for a short while, then reverts back to the original settings.

After poking around some more, I found that it does this for all projects - I created an empty folder and added it to codekit as a project, just to make sure it wasn't anything wrong with one particular project.

I also found that the default project settings are ignored, and that whenever quitting codekit, they will always revert back to the factory default CoffeeLint settings. This isn't an issue for any of the other linters - just CoffeeLint.

It seems that any project-specific changes aren't being saved in the config.codekit file, and hand editing it (a poor life decision) worked for me as a temporary fix.

tl;dr
- Project-specific CoffeeLint settings aren't saved
- User's default CoffeeLint settings are ignored in favor of factory defaults

Running OSX 10.9.5, CodeKit 2.1.6

**Comment by bdkjones** [comment]
Well holy crap, you’re right. I’ll get that fixed.

**Comment by bdkjones** [comment]
Fixed for 2.1.7. I'm trying to get that out later today. Have to decide how to deal with the Foundation/Sass/Libsass disaster.
